SwaMeTest/SwaMeTest/ChromatogramMetricGeneratorTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using MzmlParser;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace SwaMe.Test
+{
+    [TestClass]
+    public class MetricGeneratorTest
+    {
+        private static Run run;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+
+            run = new Run();
+            run.AnalysisSettings = new AnalysisSettings
+            {
+                RtTolerance = 2.5
+            };
+
+
+            var basePeak1 = new BasePeak(150, 2.5, 150);
+            basePeak1.BpkRTs = new List<double>() { 2.58 };
+            var spectrumpoint1 = new SpectrumPoint();
+            spectrumpoint1.Intensity = 2000;
+            spectrumpoint1.Mz = 150;
+            spectrumpoint1.RetentionTime = 2.58F;
+            var spectrumpoint2 = new SpectrumPoint();
+            spectrumpoint2.Intensity = 3000;
+            spectrumpoint2.Mz = 150.01F;
+            spectrumpoint2.RetentionTime = 3.00F;
+
+            basePeak1.Spectrum = new List<SpectrumPoint>() { spectrumpoint1, spectrumpoint2 };
+
+            run.BasePeaks = new ConcurrentBag<BasePeak>() { basePeak1};
+        }
+               
+
+        [TestMethod]
+        public void CheckIntensitiesAndStartTimesIfOneBasePeak()
+        {
+            var sC = new ChromatogramMetricGenerator();
+            sC.GenerateChromatogram(run);
+            var correctIntFile = File.ReadAllText("..\\..\\..\\..\\correctIntensities.csv");

```

Comment:
- Any file reading should be done in the initialise, test should have minimum content
- Right click the test file and set it to copy to output directory, don't do all this directory navigation
- Backslashes will fail on Linux, so use Path.Combine to concat directories and files (this will use the correct separator for the environment)

Reply:
I think we might be doing this a backwards - for the other tests we're putting test data into an external file, loading that in, then the data we're asserting against is essentially hard-coded into our code.

Here, it appears that the tests data is hard-coded and the correct answers are in the file.